x-pack/plugin/watcher/src/main/java/org/elasticsearch/xpack/watcher/execution/ExecutionService.java

```diff
@@ -318,7 +318,8 @@ public WatchRecord execute(WatchExecutionContext ctx) {
                             historyStore.put(record);
                         }
                     } catch (Exception e) {
-                        logger.error((Supplier<?>) () -> new ParameterizedMessage("failed to update watch record [{}]", ctx.id()), e);
+                        logger.error((Supplier<?>) () -> new ParameterizedMessage("failed to update watch status[{}]",
+                            ctx.id().watchId()), e);
```

Comment:
I'm not sure I follow this change.  
* record vs. status -  isn't it the record (watcher history) that fails to update, not sure what status means here
* ctx.id vs. ctx.id.watchId - doesn't ctx.id include the watch id + execution time, which is what we want here ?